bin/circle-functions

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+install_wordepress() {
```

Comment:
It seems to me this would work equally well as a script you execute (rather than a function you source).
This would save 2 lines in this file and 2 each time it is called.